### PR TITLE
docs: recipe for nextjs

### LINKED ISSUE #688: Add `next.js` in `Docs/Receipts/CRA, Vite, Webpack or Other`
A warning of `fs` and `crypto` was introduced and the solution was mentioned at https://lumos-website.vercel.app/recipes/cra-vite-webpack-or-other

But a popular framework [Next.js](https://nextjs.org/) is not on the list.

I'd suggest adding code snippet for Next.js so users don't have to figure it out by themselves.

Ref:
- Custom Webpack Config in Next.js: https://nextjs.org/docs/app/api-reference/next-config-js/webpack

### PR DESCRIPTION
**Pull request by homura**
# Description

Fixes #688

This PR added an example for working with Next.js

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation or Website

### CODE REVIEW

File: packages/hd/src/xpub_store.ts [hunk 1]
```diff
@@ -12,28 +11,13 @@ export class XPubStore {
...
-
   toJson(): string {
     return JSON.stringify({
       xpubkey: this.accountExtendedPublicKey.serialize().slice(2),
     });
   }
 
-  static load(path: string): XPubStore {
-    const json = fs.readFileSync(path, "utf-8");
+  static loadFromJson(json: string): XPubStore {
```
Comment (twhy) [inline packages/hd/src/xpub_store.ts]: Consider just using `fromJson` for this method name. 

`toJson` and `fromJson` makes a good pair.

**Review by twhy — APPROVED** [review]

File: packages/hd/src/xpub_store.ts (hunk 1 above)
Comment (twhy) [reply packages/hd/src/xpub_store.ts]: @homura  Please reslove this.

**Comment by Keith-CY** [comment]
This PR includes functional updates but the title starts with `docs:`, which will be misleading after merge.

**Review by Keith-CY — APPROVED** [review]

**Comment by homura** [comment]
> This PR includes functional updates but the title starts with `docs:`, which will be misleading after merge.

I've separated the codecov upgrade and dropped fs support for xpub store into other PRs

**Pull request merged**